scripts/build/build/targets.py

```diff
@@ -254,7 +257,7 @@ def HostTargets():
     builder.AppendVariant(name="test-group", validator=AcceptNameWithSubstrings(
         ['-all-clusters', '-chip-tool']), test_group=True),
     builder.AppendVariant(name="same-event-loop", validator=AcceptNameWithSubstrings(
-        ['-chip-tool']), separate_event_loop=False),
+        ['-chip-tool', '-chip-tool-darwin']), separate_event_loop=False),
```

Comment:
`darwin-chip-tool-darwin` stutters and is very easy to mistake for `darwin-chip-tool`. Are there other naming options?

Reply:
I am ok with any name that does not share a full prefix with another - I believe that is what bothers me.

I imagine we can do changes in several steps: whatever you name it here, the output will still be called chip-tool I believe which is annoying. If you want this change as-is and followup for rename that is fine as well, just let me know.

I am not blocking the PR on this, just mentioning since reusing is really confusing. I think I already asked `why darwin-chip-tool and chip-tool-darwin are different things` before and that is my instinct still whenever I see this naming. Thinking I am not the only one.

Reply:
Ok. We will use this [issue](https://github.com/project-chip/connectedhomeip/issues/14987) to do a refactor on renaming chip-tool-darwin as a follow up.